2016/figures.html

```diff
@@ -0,0 +1,191 @@
+      Focus groups, Facebook ads, data investigations and on-the-ground visits help us ensure we’re <a href="https://www.mysociety.org/research/">actually making a difference</a>.
+    </p>
+  </div>
+
+  <p>This year, we <a href="https://openknowledge.worldbank.org/handle/10986/23752">evaluated citizen engagement</a> and <a href="https://www.mysociety.org/2016/02/19/our-latest-research-citizen-engagement-in-new-zealand-and-australia/">investigated civic initiatives in New Zealand and Australia</a>.</p>
+  <p>But we also conducted targeted research in FOI, Democracy, and Better Cities - more details in those sections.</p>
+
+</div>
+
+<div class="section section--no-padding">
+  <img src="{{ site.baseurl }}/assets/img/tictec-blue.jpg" alt="" style="width: 100%;">
+</div>
+
+<div class="section section--white section--figures-tictec">
+
+  <img src="{{ site.baseurl }}/assets/img/quote-tictec.png" srcset="{{ site.baseurl }}/assets/img/quote-tictec@2x.png 2x" alt="What's the favourite thing you've worked on this year? Gemma (Events Manager): Producing TICTeC 2016 in Barcelona was the absolute highlight for me this year. Not only is Barcelona an awesome city to work in but the people who came to TICTeC seemed to genuinely love it, which is the best feeling as an event organiser. " class="quote quote--tictec">
+
+  <div class="stat">
+    <p class="stat__main">
+      <span class="stat__number">136</span>
+      <span class="stat__name">people attended TICTeC 2016</span>
+    </p>
+    <p class="stat__comment">
+      <em>¡Viva la tecnologia cívica!</em> Our annual conference on the Impacts of Civic Technology was <a href="https://www.mysociety.org/research/tictec-2016/">held in Barcelona this year</a>. Churros and civic tech are a great mix.
+    </p>
+  </div>
+
+  <div class="stat">
+    <p class="stat__main">
+      <span class="stat__number">5</span>
+      <span class="stat__name">UK councils partnered with</span>
+    </p>
+    <p class="stat__comment">
+      Our <a href="https://www.fixmystreet.com/about/council">FixMyStreet for Councils</a> product is better by magnitudes, thanks to the councils who generously gave us their insights and experience.
+    </p>
+  </div>
+
+</div>
+
+<div class="section section--figures-team">
+  <div class="row">
+    <div class="column-one-half">
+
+      <img src="{{ site.baseurl }}/assets/img/quote-snacks.png" srcset="{{ site.baseurl }}/assets/img/quote-snacks@2x.png 2x" alt="What do you like best about working at mySociety? Martin (Designer): No peer pressure to manage my snack intake! And what do you like least about working at mySociety? Martin: No peer pressure to manage my snack intake [sad face]" class="quote quote--snacks">
+
+    </div>
+    <div class="column-one-half">
+
+      <div class="stat-container">
+        <div class="stat">
+          <p class="stat__main">
+            <span class="stat__number">17</span>
+            <span class="stat__name">team member towns</span>
```

Comment:
We forgot the change in Martin's speech bubbles: "What do you like best about working from home?"